pkg/cmd/secret/set/http.go

```diff
@@ -75,10 +85,14 @@ func putOrgSecret(client *api.Client, host string, pk *PubKey, opts SetOptions,
 	orgName := opts.OrgName
 	visibility := opts.Visibility
 
-	var repositoryIDs []int
+	var repositoryIDs []string
 	var err error
 	if orgName != "" && visibility == shared.Selected {
-		repositoryIDs, err = mapRepoNameToID(client, host, orgName, opts.RepositoryNames)
+		repos := make([]ghrepo.Interface, 0, len(opts.RepositoryNames))
+		for _, repo := range opts.RepositoryNames {
+			repos = append(repos, ghrepo.New(opts.OrgName, repo))
```

Comment:
Since we now accept fully-qualified repo names when setting Codespaces secrets, let's also (optionally) accept fully qualified names when setting org secrets too. Of course, accepting just repo names should be kept for backwards compatibility.

```
gh secret set -o acme -r acme/project1,acme/project2
```

Reply:
Added with https://github.com/cli/cli/pull/4699/commits/e0d9b32d28b0bcbee91cc89085c7586a2db45f99